### PR TITLE
Create default visual for products with no picture #1322

### PR DESCRIPTION
**Pull request by ithiame**

### CODE REVIEW

File: src/lib/components/ProductWidget/ProductWidgetPOS.svelte [hunk 1]
```diff
@@ -13,7 +14,11 @@
 <div class="touchScreen-product-cta flex flex-row {className}">
 	<div>
 		<a href="/product/{product._id}">
-			<PictureComponent picture={pictures[0]} class="object-contain h-24 w-24" />
+			{#if pictures.length > 0 && pictures[0]}
+				<PictureComponent picture={pictures[0]} class="object-contain h-24 w-24" />
+			{:else}
+				<img class="object-contain h-24 w-48" src={DEFAULT_PICTURE} alt="default-img" />
```
Comment (coyotte508) [inline src/lib/components/ProductWidget/ProductWidgetPOS.svelte:20]:
```suggestion
				<img class="object-contain h-24 w-48" src={DEFAULT_PICTURE} alt="" />
```

**Review by coyotte508 — APPROVED** [review]

File: src/lib/components/ProductWidget/ProductWidgetPOS.svelte (hunk 1 above)
Comment (ithiame) [reply src/lib/components/ProductWidget/ProductWidgetPOS.svelte:20]: @coyotte508 sometime the image doesn't exist in the S3
image

Is there a function or something you already implemented that check if the image exist int the S3 ?

Comment (coyotte508) [reply src/lib/components/ProductWidget/ProductWidgetPOS.svelte:20]: well it should always exist. You can add event handlers in `Picture` (eg `on:error`) or a fallback picture. But it should always exist...

Comment (Tirodem) [reply src/lib/components/ProductWidget/ProductWidgetPOS.svelte:20]: Functionally, it should, but as said earlier, I want the /pos/touch interface to be used even with a broken/down S3.
In this particular case, it's a remainder of import / export feature testing, but that's what gave me the idea of a fallback picture.
And, in the future, maybe we'll allow product creation with no picture for restaurants particular usage, so...
(Here's for the context)

File: src/lib/assets/default-product.png
Comment (coyotte508) [inline src/lib/assets/default-product.png:1]: Can we use a SVG image instead?

It should be much more lightweight for such a simple image

**Review by coyotte508 — APPROVED** [review]

**Pull request merged**